Example/ViewController.swift

```diff
@@ -1,9 +1,88 @@
 import UIKit
+import MiniApp
 
-class ViewController: UIViewController {
+class ViewController: UITableViewController {
+
+    var decodeResponse: [MiniAppInfo]?
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        showProgressIndicator()
+
+        MiniApp.list { (result) in
+            switch result {
+            case .success(let responseData):
+                self.decodeResponse = responseData
+                self.tableView.reloadData()
+                self.dismiss(animated: false, completion: nil)
+            case .failure(let error):
+                print(error.localizedDescription)
+                self.displayAlert(title: "Error", message: "Couldn't retrieve Mini App list, please try again later")
+            }
+        }
+    }
+
+    func decodeListingResponse(with dataResponse: Data?) -> [MiniAppInfo]? {
```

Comment:
I don't see anyplace where this function is used. Seems like something that shouldn't be needed?

Reply:
Yes. Removed them now